variances.scala:2: error: covariant type A occurs in contravariant position in type Vector[A] of value x
  def append(x: Vector[A]): Vector[A] 
             ^
one error found
